src/pages/security/tor/index.md

```diff
@@ -1,20 +1,24 @@
 ---
-title: Tor (The Onion Router)
+title: Tor, The Onion Router
```

Comment:
I'm not sure whether this change is needed. The style guide suggests using the full name first, with the acronym in brackets.

Reply:
@fvcproductions what do you think of this?

Reply:
@jp-sauve I think should probably be changed to `Tor` as per style guidelines. 👍 

Also, we should make sure the changes in this PR are significant/substantial enough to merit a merge.